### ISSUE TITLE
[4.0.Neutronium] Header from previous page on navigate back

### ISSUE THREAD

**Issue by ludufre** [ionic-team/ionic#17297]
**Ionic version:**

[x] **4.0 Neutronium**

**I'm submitting a ...** 

[x] bug report
[ ] feature request

**Current behavior:**

When I return from a page, the header of the previous page appears for some milliseconds and disappears.

**Expected behavior:**

Do not display the header of the previous page on the current page even if it is for a few milliseconds.

**Steps to reproduce:**

**Related code:**

```
ionic start bug blank
ionic cordova platform add ios
ionic g page example
```
add `<a routerLink="/example">Go to Example</a>` to `ion-content` in home.page.html

add `<ion-buttons slot="start"><ion-back-button></ion-back-button></ion-buttons>` to `ion-toolbar` in example.page.html

```
ionic cordova run ios
```
Click in `Go to Example` then, click in back arrow.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/4112077/51862940-e9d09080-2326-11e9-982c-6faaf3ce7a1b.gif)

**Other information:**

Example repository: https://github.com/ludufre/ionic-v4-back-bug

**Ionic info:** 

```
Ionic:

   ionic (Ionic CLI)             : 4.9.0 (/Users/ludufre/.npm-packages/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0
   @angular-devkit/build-angular : 0.12.3
   @angular-devkit/schematics    : 7.2.3
   @angular/cli                  : 7.2.3
   @ionic/angular-toolkit        : 1.2.3

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.2, (and 4 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/ludufre/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 7.0.0
   NodeJS            : v10.15.0 (/usr/local/bin/node)
   npm               : 6.7.0
   OS                : macOS Mojave
   Xcode             : Xcode 10.1 Build version 10B61
```

**Comment by dromedaryCase** [comment]
This looks identical to #17022 and #16988

**Comment by dromedaryCase** [comment]
Don't know. This bug is such an immersion breaker for every native build. It immediately gives away, that the user is looking at a non-native app.

I find your issue description the most ... erm ... descriptive so far ^^

I'm looking forward to see, if #17224 has the desired side effect of fixing this.

**Comment by ludufre** [comment]
Still not fixed yet in 4.0.1 update.

**Comment by armansik** [comment]
Still not fixed yet in 4.0.2 update.

**Comment by brandyscarney** [comment]
Closing as a duplicate of #16988 please track this issue there, thanks!
